### PR TITLE
move images to drop images/en folder

### PR DESCRIPTION
**Pull request by samthor**
I was experimenting with trying to host our content on Firebase Hosting, but realised that we still had two top-level image directories that are effectively munged together.

I don't see the reason behind this—none of the images contain a ton of English content and in some cases (icons/) we just had duplicate icons.

Notes:
- I updated three articles that referenced shared but I triggered the "updated:" must change logic (even though it's the same article, we're just swapping an image)
- I added compression to our top-level images, but just for JPEGs (author images)
  - I know we're moving to imgix for a bunch of stuff so perhaps we don't need this at all
- I cleared a bunch of exec bits on the user images (since why?)

### CODE REVIEW

**Comment by robdodson** [comment]
btw this broke the /authors/ page

image

If percy is failing be sure to give it a glance. We can keep sorting out the issue with why some images are failing to load on /learn/ but we shouldn't ignore the failures.

**Comment by robdodson** [comment]
oh! this hasn't been merged yet! 😅 Sorry for the previous comment, I thought you had self-merged, my mistake.

File: build.js [hunk 1]
```diff
@@ -63,20 +63,24 @@ const defaultPlugins = [
 async function buildCacheManifest() {
   const toplevelManifest = await getManifest({
     globDirectory: 'dist',
-    globPatterns: ['images/**', '*.css', '*.js'],
+    globPatterns: [
+      'images/**/*.{png,svg}', // .jpg files are used for authors, skip
```
[the remark this answers is not in the document]
Comment (samthor) [reply build.js:74]: Done

**Comment by samthor** [comment]
Nice catch on the "en" for author images. @MichaelSolati your PR might either need that change, or I'll need to make the change, depending on who gets to submit first.

**Comment by samthor** [comment]
I've updated this PR as @MichaelSolati's PR won.

**Comment by robdodson** [comment]
hm is the authors page still broken?

**Comment by samthor** [comment]
@robdodson Percy is happy now.

**Pull request merged**